### ISSUE TITLE
Xpra 5.0 fails to render menu windows for JetBrains IDEs

### ISSUE THREAD

**Issue by jzyrobert** [Xpra-org/xpra#3891]
**Describe the bug**
When trying to use a JetBrains IDE with the latest Xpra 5.0 Beta versions from Windows to Ubuntu 22, the "Open projects" window fails to show, with the specific error being
```
2023-06-09 11:39:30,818 client   1 @18.530 failed to instantiate <class 'xpra.client.gtk3.client_window.ClientWindow'>
2023-06-09 11:39:30,818 client   1 @18.530   File "E:/xpra/xpra/client/mixins/window_manager.py", line 853, in make_new_window
2023-06-09 11:39:30,818 client   1 @18.530   File "E:/xpra/xpra/client/gui/client_window_base.py", line 95, in __init__
2023-06-09 11:39:30,819 client   1 @18.530   File "E:/xpra/xpra/client/gtk3/gtk_client_window_base.py", line 719, in finalize_window
2023-06-09 11:39:30,819 client   1 @18.530   File "E:/xpra/xpra/client/gtk3/gtk_client_window_base.py", line 729, in setup_following
2023-06-09 11:39:30,819 client   1 @18.530 TypeError: argument of type 'NoneType' is not iterable
2023-06-09 11:39:30,819 client   1 @18.530 no more options.. this window will not be shown, sorry
```

**To Reproduce**
1. `xpra start :100 --open-files=no --min-speed=80 --min-quality=30 --bandwidth-detection=no --no-daemon -d window`
2. `Xpra_cmd.exe attach ssh://user@host/100 --notifications=no --speaker=no --printing=no --opengl=no`
3. Launch a latest release version of JetBrains Goland or Rider (2023.1.2) and try to press the "Open" button to open a project.

**System Information (please complete the following information):**
 - Server OS: Ubuntu 22.04
 - Client OS: Windows 10
 - Xpra Server Version: 5.0-r33635 (Latest as of 06/06)
 - Xpra Client Version: 5.0-r33624 (Latest as of 06/06)

**Additional context**
This works fine in version `4.4.5`
When using `-d window` on the server, the full output is
```
2023-06-09 11:39:30,813 Warning: found large packet
2023-06-09 11:39:30,814  'new-window' packet is 4840 bytes:
2023-06-09 11:39:30,814  argument types: <class 'int'>, <class 'int'>, <class 'int'>, <class 'int'>, <class 'int'>, <class 'dict'>, <class 'dict'>
2023-06-09 11:39:30,814  sizes: 1, 4, 3, 3, 3, 4980, 2
2023-06-09 11:39:30,814  packet: [30, 3, 1695, 835, 424, 454, {'xid': 14680228, ' .. ed-position': (1695, 835), 'iconic': False}, {}]
2023-06-09 11:39:30,814 get_wm_state(fullscreen) state_names=('_NET_WM_STATE_FULLSCREEN',)
2023-06-09 11:39:30,815 get(fullscreen, None) using get_property=False
2023-06-09 11:39:30,815 get(content-type, ) using get_property=text
2023-06-09 11:39:30,815 get(quality, -1) using get_property=-1
2023-06-09 11:39:30,815 get(speed, -1) using get_property=-1
2023-06-09 11:39:30,815 get(opaque-region, ()) using get_property=()
2023-06-09 11:39:30,815 initial encoding for 3: auto
2023-06-09 11:39:30,815 insert_encoder(enc_rgb, rgb24, <function encode at 0x7f84ee440790>)
2023-06-09 11:39:30,815 insert_encoder(enc_rgb, rgb32, <function encode at 0x7f84ee440790>)
2023-06-09 11:39:30,815 insert_encoder(enc_pillow, png, <function encode at 0x7f84ee1a5f30>)
2023-06-09 11:39:30,815 insert_encoder(enc_pillow, png/L, <function encode at 0x7f84ee1a5f30>)
2023-06-09 11:39:30,815 insert_encoder(enc_pillow, png/P, <function encode at 0x7f84ee1a5f30>)
2023-06-09 11:39:30,815 insert_encoder(enc_pillow, jpeg, <function encode at 0x7f84ee1a5f30>)
2023-06-09 11:39:30,815 insert_encoder(enc_pillow, webp, <function encode at 0x7f84ee1a5f30>)
2023-06-09 11:39:30,815 insert_encoder(enc_webp, webp, <built-in function encode>)
2023-06-09 11:39:30,815 insert_encoder(enc_jpeg, jpeg, <built-in function encode>)
2023-06-09 11:39:30,815 insert_encoder(enc_jpeg, jpega, <built-in function encode>)
2023-06-09 11:39:30,815 insert_encoder(enc_avif, avif, <built-in function encode>)
2023-06-09 11:39:30,815 init_encoders() cuda_device_context=None
2023-06-09 11:39:30,815 parse_csc_modes({b'h264': (b'ARGB', b'BGRA', b'BGRX', b'GBRP', b'GBRP9LE', b'RGB', b'XRGB', b'YUV420P', b'YUV422P', b'YUV444P'), b'vp8': (b'YUV420P',), b'h265': (b'BGRX', b'GBRP', b'GBRP9LE', b'RGB', b'XRGB', b'YUV420P', b'YUV422P', b'YUV444P'), b'mpeg4': (b'YUV420P',), b'vp9': (b'YUV420P', b'YUV444P'), b'webp': (b'BGRX', b'BGRA'), b'jpeg': (b'BGRX', b'BGRA', b'YUV420P'), b'jpega': (b'BGRA', b'RGBA')}) current value=typedict({})
2023-06-09 11:39:30,815 ws.update_encoding_selection(auto, [], True) encoding=auto, common encodings=('png', 'png/P', 'png/L', 'webp', 'avif', 'rgb24', 'rgb32', 'jpeg', 'jpega')
2023-06-09 11:39:30,816 update_encoding_selection: client refresh encodings=(), auto_refresh_encodings=('png', 'webp', 'jpega')
2023-06-09 11:39:30,816 update_encoding_options(False) wid=3, want_alpha=False, speed=80, quality=100
2023-06-09 11:39:30,816 lossless threshold: 76 / 5, rgb auto threshold=3072 (min=2048, max=16384)
2023-06-09 11:39:30,816 bandwidth-limit=0, get_best_encoding=<bound method WindowSource.get_auto_encoding of WindowVideoSource(3 : (424, 454))>
2023-06-09 11:39:30,816 update_refresh_attributes() wid=3, sizef=0.44, content-type=text, qf=0.50, sf=0.70, cf=1.00, batch delay=46, bandwidth-limit=0, min-delay=184, max-delay=1000, delay=184
2023-06-09 11:39:30,816 do_set_auto_refresh_delay(184, 184)
2023-06-09 11:39:30,816 update_refresh_attributes() wid=3, refresh quality=100%, refresh speed=50%, for cv=0.00, bwl=0
2023-06-09 11:39:30,816 append_encoder(h264, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 append_encoder(vp9, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 append_encoder(vp8, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 append_encoder(webp, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 append_encoder(jpeg, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 append_encoder(jpega, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 insert_encoder(auto, auto, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 insert_encoder(stream, stream, <bound method WindowVideoSource.video_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 insert_encoder(scroll, scroll, <bound method WindowVideoSource.scroll_encode of WindowVideoSource(3 : (424, 454))>)
2023-06-09 11:39:30,816 _update_client_geometry: using client-geometry=(1695, 835, 424, 454)
2023-06-09 11:39:30,816 _do_update_client_geometry((1695, 835, 424, 454))
2023-06-09 11:39:30,816 calc_constrained_size(424, 454, typedict({'position': (1695, 835), 'gravity': 1, 'minimum-size': (136, 39)}))=(424, 454) (size_constraints=(136, 39, 32767, 32767))
2023-06-09 11:39:30,816 _do_update_client_geometry: size({'position': (1695, 835), 'gravity': 1, 'minimum-size': (136, 39)})=424x454
2023-06-09 11:39:30,816 updateprop(geometry, (1695, 835, 424, 454)) unchanged
2023-06-09 11:39:30,818 client   1 @18.530 failed to instantiate <class 'xpra.client.gtk3.client_window.ClientWindow'>
2023-06-09 11:39:30,818 _update_client_geometry: using client-geometry=(1695, 835, 424, 454)
2023-06-09 11:39:30,818 client   1 @18.530   File "E:/xpra/xpra/client/mixins/window_manager.py", line 853, in make_new_window
2023-06-09 11:39:30,818 client   1 @18.530   File "E:/xpra/xpra/client/gui/client_window_base.py", line 95, in __init__
2023-06-09 11:39:30,819 client   1 @18.530   File "E:/xpra/xpra/client/gtk3/gtk_client_window_base.py", line 719, in finalize_window
2023-06-09 11:39:30,819 client   1 @18.530   File "E:/xpra/xpra/client/gtk3/gtk_client_window_base.py", line 729, in setup_following
2023-06-09 11:39:30,819 client   1 @18.530 TypeError: argument of type 'NoneType' is not iterable
2023-06-09 11:39:30,818 _do_update_client_geometry((1695, 835, 424, 454))
2023-06-09 11:39:30,819 calc_constrained_size(424, 454, typedict({'position': (1695, 835), 'gravity': 1, 'minimum-size': (136, 39)}))=(424, 454) (size_constraints=(136, 39, 32767, 32767))
2023-06-09 11:39:30,819 _do_update_client_geometry: size({'position': (1695, 835), 'gravity': 1, 'minimum-size': (136, 39)})=424x454
2023-06-09 11:39:30,819 updateprop(geometry, (1695, 835, 424, 454)) unchanged
2023-06-09 11:39:30,819 client   1 @18.530 no more options.. this window will not be shown, sorry
2023-06-09 11:39:30,819 Property changed on 0xe000a4: _NET_WM_ALLOWED_ACTIONS
2023-06-09 11:39:30,820 Property changed on 0xe000a4: _NET_FRAME_EXTENTS
2023-06-09 11:39:30,820 Property changed on 0xe000a4: _NET_WM_STATE
2023-06-09 11:39:30,820 Property changed on 0xe000a4: WM_STATE
2023-06-09 11:39:30,820 Property changed on 0xe000a4: WM_TRANSIENT_FOR
2023-06-09 11:39:30,820 WM_TRANSIENT_FOR=14680140
2023-06-09 11:39:30,820 updateprop(transient-for, <GdkX11.X11Window object at 0x7f84c10e8a80 (GdkX11Window at 0x5560cd7b73e0)>) unchanged
2023-06-09 11:39:30,821 invalidating named pixmap, contents handle=None
2023-06-09 11:39:30,821 Property changed on 0xe000a4: WM_NORMAL_HINTS
2023-06-09 11:39:30,821 WM_NORMAL_HINTS={'position': (1687, 804), 'size': (424, 454), 'min_size': (272, 143), 'win_gravity': 1}
2023-06-09 11:39:30,821 get(decorations, -1) using get_property=30
2023-06-09 11:39:30,821 updateprop(size-hints, {'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}) previous value={'position': (1695, 835), 'gravity': 1, 'minimum-size': (136, 39)}
2023-06-09 11:39:30,821 updated: size-hints={'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}
2023-06-09 11:39:30,821 _update_client_geometry: using client-geometry=(1695, 835, 424, 454)
2023-06-09 11:39:30,821 _do_update_client_geometry((1695, 835, 424, 454))
2023-06-09 11:39:30,821 calc_constrained_size(424, 454, typedict({'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}))=(424, 454) (size_constraints=(136, 39, 32767, 32767))
2023-06-09 11:39:30,821 _do_update_client_geometry: size({'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)})=424x454
2023-06-09 11:39:30,821 updateprop(geometry, (1695, 835, 424, 454)) unchanged
2023-06-09 11:39:30,822 do_child_configure_request_event(<X11:ConfigureRequest {'serial': 'd36', 'delivered_to': '400037', 'window': 'e000a4', 'x': '1695', 'y': '835', 'width': '424', 'height': '454', 'border_width': '0', 'above': '0', 'detail': '0', 'value_mask': '15'}>) client=0xe000a4, corral=0x400037, value_mask=X|Y|Width|Height, size-hints={'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}
2023-06-09 11:39:30,822 updateprop(geometry, (1695, 835, 424, 454)) unchanged
2023-06-09 11:39:30,822 do_child_configure_request_event updated requested geometry from (1695, 835, 424, 454) to  (1695, 835, 424, 454)
2023-06-09 11:39:30,823 do_damage(0, 0, 424, 454, {})     wid=3, scheduling batching expiry for sequence    1 in   0 ms
2023-06-09 11:39:30,823  delay=0, elapsed=671873129, resize_elapsed=671873129, congestion_elapsed=671873129, batch=46, min=15, inc=0
2023-06-09 11:39:30,823 expire_delayed_region(671873, 46) delayed region=DelayedRegion(time=671873, expired=False, encoding=auto, regions=[R(0, 0, 424, 454)], options={})
2023-06-09 11:39:30,823 send_delayed for wid 3, batch delay is 46ms, elapsed time is 0ms
2023-06-09 11:39:30,823 do_send_delayed() damage delayed=DelayedRegion(time=671873, expired=True, encoding=auto, regions=[R(0, 0, 424, 454)], options={})
2023-06-09 11:39:30,823 acknowledge_changes() xshm handle=None, damage handle=4194358
2023-06-09 11:39:30,823 invalidating named pixmap, contents handle=None
2023-06-09 11:39:30,823 send_delayed_regions: using full window update 424x454 as  webp: merged region covers almost the whole window, from <bound method WindowSource.get_auto_encoding of WindowVideoSource(3 : (424, 454))>
2023-06-09 11:39:30,823 refreshing named pixmap
2023-06-09 11:39:30,824 scroll detection took 0ms
2023-06-09 11:39:30,825 make_data_packet: image=XShmImageWrapper(BGRX: 0, 0, 424, 454), damage data: (3, 0, 0, 424, 454, 'webp')
2023-06-09 11:39:30,826 compress:   1.7ms for  424x454  pixels at    0,0    for wid=3     using      webp with ratio   0.0%  (  752KB to     1KB), sequence     1, client_options={'rgb_format': 'BGRX', 'quality': 100, 'encoder': 'webp'}, options={'quality': 100, 'speed': 80, 'rgb_formats': ('RGB', 'RGBX', 'RGBA'), 'lz4': True, 'content-type': 'text', 'cuda-device-context': None}
2023-06-09 11:39:30,827 auto refresh:  webp screen update (actual quality=100, lossy=False), lossless - nothing to do (region=rectangle(0, 0, 424, 454), refresh regions=[], exclude=None)
2023-06-09 11:39:30,827 Property changed on 0xe000a4: _XPRA_WID
2023-06-09 11:39:30,827 _handle_property_change(_XPRA_WID) property type=(b'CARDINAL', 32)
2023-06-09 11:39:30,827 _handle_property_change(_XPRA_WID) value=3
2023-06-09 11:39:30,827 Property changed on 0xe000a4: WM_NORMAL_HINTS
2023-06-09 11:39:30,828 WM_NORMAL_HINTS={'position': (1687, 804), 'size': (424, 454), 'min_size': (272, 143), 'win_gravity': 1}
2023-06-09 11:39:30,828 get(decorations, -1) using get_property=30
2023-06-09 11:39:30,828 updateprop(size-hints, {'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}) unchanged
2023-06-09 11:39:30,828 do_xpra_child_map_event(<X11:MapNotify {'serial': 'd56', 'delivered_to': '50d', 'window': '400037', 'override_redirect': 'False'}>)
2023-06-09 11:39:30,828 damage(0, 0, 424, 454, {}) wid=3
2023-06-09 11:39:30,828 do_damage(0, 0, 424, 454, {})     wid=3, scheduling batching expiry for sequence    2 in  41 ms
2023-06-09 11:39:30,828  delay=41, elapsed=5, resize_elapsed=671873135, congestion_elapsed=671873135, batch=46, min=15, inc=0
2023-06-09 11:39:30,828 WindowModel.do_xpra_configure_event(<X11:ConfigureNotify {'serial': 'd78', 'delivered_to': 'e000a4', 'window': 'e000a4', 'x': '1695', 'y': '835', 'width': '424', 'height': '454', 'border_width': '0', 'above': '0'}>) corral=0x400037, client=0xe000a4, managed=True
2023-06-09 11:39:30,829 calc_constrained_size(424, 454, typedict({'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}))=(424, 454) (size_constraints=(136, 39, 32767, 32767))
2023-06-09 11:39:30,829 resize_corral_window(1695, 835, 424, 454) hints={'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}, constrained size=(424, 454), geometry=(1695, 835, 424, 454), resized=False, moved=True
2023-06-09 11:39:30,829 updateprop(geometry, (1695, 835, 424, 454)) unchanged
2023-06-09 11:39:30,829 invalidating named pixmap, contents handle=PixmapWrapper(0x400038, 424, 454)
2023-06-09 11:39:30,830 damage(0, 0, 424, 454, {}) wid=3
2023-06-09 11:39:30,830 do_damage(0, 0, 424, 454, {})     wid=3, using existing 1 delayed regions created 0.0ms ago
2023-06-09 11:39:30,830 WindowModel.do_xpra_configure_event(<X11:ConfigureNotify {'serial': 'd9a', 'delivered_to': 'e000a4', 'window': 'e000a4', 'x': '0', 'y': '0', 'width': '424', 'height': '454', 'border_width': '0', 'above': '0'}>) corral=0x400037, client=0xe000a4, managed=True
2023-06-09 11:39:30,830 calc_constrained_size(424, 454, typedict({'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}))=(424, 454) (size_constraints=(136, 39, 32767, 32767))
2023-06-09 11:39:30,830 resize_corral_window(0, 0, 424, 454) hints={'position': (1687, 804), 'gravity': 1, 'minimum-size': (272, 143)}, constrained size=(424, 454), geometry=(1695, 835, 424, 454), resized=False, moved=False
2023-06-09 11:39:30,830 invalidating named pixmap, contents handle=None
2023-06-09 11:39:30,830 Warning: client cannot find window 3
2023-06-09 11:39:30,870 expire_delayed_region(671873, 46) delayed region=DelayedRegion(time=671873, expired=False, encoding=auto, regions=[R(0, 0, 424, 454)], options={})
2023-06-09 11:39:30,870 send_delayed for wid 3, batch delay is 46ms, elapsed time is 41ms
2023-06-09 11:39:30,870 do_send_delayed() damage delayed=DelayedRegion(time=671873, expired=True, encoding=auto, regions=[R(0, 0, 424, 454)], options={})
2023-06-09 11:39:30,870 acknowledge_changes() xshm handle=XShmWrapper(0xe000a4 - 424x454), damage handle=4194358
2023-06-09 11:39:30,871 invalidating named pixmap, contents handle=None
2023-06-09 11:39:30,871 send_delayed_regions: using full window update 424x454 as  webp: merged region covers almost the whole window, from <bound method WindowSource.get_auto_encoding of WindowVideoSource(3 : (424, 454))>
2023-06-09 11:39:30,871 refreshing named pixmap
2023-06-09 11:39:30,872 scroll detection took 0ms
2023-06-09 11:39:30,872 make_data_packet: image=XShmImageWrapper(BGRX: 0, 0, 424, 454), damage data: (3, 0, 0, 424, 454, 'webp')
2023-06-09 11:39:30,878 compress:   6.5ms for  424x454  pixels at    0,0    for wid=3     using      webp with ratio   7.8%  (  752KB to    59KB), sequence     2, client_options={'rgb_format': 'BGRX', 'quality': 100, 'encoder': 'webp'}, options={'quality': 100, 'speed': 80, 'rgb_formats': ('RGB', 'RGBX', 'RGBA'), 'lz4': True, 'content-type': 'text', 'cuda-device-context': None}
2023-06-09 11:39:30,879 damage(13, 58, 398, 336, {}) wid=3
2023-06-09 11:39:30,879 do_damage(13, 58, 398, 336, {})   wid=3, scheduling batching expiry for sequence    3 in  39 ms
2023-06-09 11:39:30,879  delay=38, elapsed=8, resize_elapsed=671873186, congestion_elapsed=671873186, batch=46, min=15, inc=1
2023-06-09 11:39:30,880 auto refresh:  webp screen update (actual quality=100, lossy=False), lossless - nothing to do (region=rectangle(0, 0, 424, 454), refresh regions=[], exclude=None)
2023-06-09 11:39:30,881 damage(250, 416, 78, 30, {}) wid=3
2023-06-09 11:39:30,881 do_damage(250, 416, 78, 30, {})   wid=3, using existing 2 delayed regions created 0.0ms ago
2023-06-09 11:39:30,882 Warning: client cannot find window 3
2023-06-09 11:39:30,891 damage(13, 394, 398, 1, {}) wid=3
2023-06-09 11:39:30,892 do_damage(13, 394, 398, 1, {})    wid=3, using existing 3 delayed regions created 0.0ms ago
2023-06-09 11:39:30,919 expire_delayed_region(671873, 46) delayed region=DelayedRegion(time=671873, expired=False, encoding=auto, regions=[R(13, 58, 398, 336), R(250, 416, 78, 30), R(13, 394, 398, 1)], options={})
2023-06-09 11:39:30,919 send_delayed for wid 3, batch delay is 46ms, elapsed time is 39ms
2023-06-09 11:39:30,919 do_send_delayed() damage delayed=DelayedRegion(time=671873, expired=True, encoding=auto, regions=[R(13, 58, 398, 336), R(250, 416, 78, 30), R(13, 394, 398, 1)], options={})
2023-06-09 11:39:30,919 acknowledge_changes() xshm handle=XShmWrapper(0xe000a4 - 424x454), damage handle=4194358
2023-06-09 11:39:30,919 invalidating named pixmap, contents handle=PixmapWrapper(0x40003a, 424, 454)
2023-06-09 11:39:30,919 send_delayed_regions: packet_cost=139538, merge_threshold=115497, pixel_count=136466
2023-06-09 11:39:30,919 send_delayed_regions: using full window update 424x454 as  webp: bytes cost (139538) too high (max 115497), from <bound method WindowSource.get_auto_encoding of WindowVideoSource(3 : (424, 454))>
2023-06-09 11:39:30,920 refreshing named pixmap
2023-06-09 11:39:30,920 scroll detection took 0ms
2023-06-09 11:39:30,920 make_data_packet: image=XShmImageWrapper(BGRX: 0, 0, 424, 454), damage data: (3, 0, 0, 424, 454, 'webp')
2023-06-09 11:39:30,925 compress:   4.5ms for  424x454  pixels at    0,0    for wid=3     using      webp with ratio   4.8%  (  752KB to    37KB), sequence     3, client_options={'rgb_format': 'BGRX', 'quality': 100, 'encoder': 'webp'}, options={'quality': 100, 'speed': 80, 'rgb_formats': ('RGB', 'RGBX', 'RGBA'), 'lz4': True, 'content-type': 'text', 'cuda-device-context': None}
2023-06-09 11:39:30,925 auto refresh:  webp screen update (actual quality=100, lossy=False), lossless - nothing to do (region=rectangle(0, 0, 424, 454), refresh regions=[], exclude=None)
2023-06-09 11:39:30,927 Warning: client cannot find window 3
2023-06-09 11:39:30,959 damage(13, 58, 398, 336, {}) wid=3
2023-06-09 11:39:30,959 do_damage(13, 58, 398, 336, {})   wid=3, scheduling batching expiry for sequence    4 in  22 ms
2023-06-09 11:39:30,959  delay=7, elapsed=39, resize_elapsed=671873266, congestion_elapsed=671873266, batch=46, min=15, inc=15
2023-06-09 11:39:30,981 expire_delayed_region(671873, 46) delayed region=DelayedRegion(time=671873, expired=False, encoding=auto, regions=[R(13, 58, 398, 336)], options={})
2023-06-09 11:39:30,982 send_delayed for wid 3, batch delay is 46ms, elapsed time is 22ms
2023-06-09 11:39:30,982 do_send_delayed() damage delayed=DelayedRegion(time=671873, expired=True, encoding=auto, regions=[R(13, 58, 398, 336)], options={})
2023-06-09 11:39:30,982 acknowledge_changes() xshm handle=XShmWrapper(0xe000a4 - 424x454), damage handle=4194358
2023-06-09 11:39:30,982 invalidating named pixmap, contents handle=PixmapWrapper(0x40003b, 424, 454)
2023-06-09 11:39:30,982 send_delayed_regions: using full frame for webp encoding of 398x336
2023-06-09 11:39:30,982 refreshing named pixmap
2023-06-09 11:39:30,983 scroll detection took 0ms
2023-06-09 11:39:30,983 make_data_packet: image=XShmImageWrapper(BGRX: 0, 0, 424, 454), damage data: (3, 0, 0, 424, 454, 'webp')
2023-06-09 11:39:30,987 compress:   4.4ms for  424x454  pixels at    0,0    for wid=3     using      webp with ratio   4.8%  (  752KB to    37KB), sequence     4, client_options={'rgb_format': 'BGRX', 'quality': 100, 'encoder': 'webp'}, options={'quality': 100, 'speed': 80, 'rgb_formats': ('RGB', 'RGBX', 'RGBA'), 'lz4': True, 'content-type': 'text', 'cuda-device-context': None}
2023-06-09 11:39:30,987 auto refresh:  webp screen update (actual quality=100, lossy=False), lossless - nothing to do (region=rectangle(0, 0, 424, 454), refresh regions=[], exclude=None)
2023-06-09 11:39:30,990 Warning: client cannot find window 3
2023-06-09 11:39:31,261 damage(12, 8, 400, 387, {}) wid=3
2023-06-09 11:39:31,261 do_damage(12, 8, 400, 387, {})    wid=3, scheduling batching expiry for sequence    5 in  10 ms
2023-06-09 11:39:31,261  delay=0, elapsed=279, resize_elapsed=671873568, congestion_elapsed=671873568, batch=46, min=15, inc=10
2023-06-09 11:39:31,271 expire_delayed_region(671873, 46) delayed region=DelayedRegion(time=671873, expired=False, encoding=auto, regions=[R(12, 8, 400, 387)], options={})
2023-06-09 11:39:31,272 send_delayed for wid 3, batch delay is 46ms, elapsed time is 10ms
2023-06-09 11:39:31,272 do_send_delayed() damage delayed=DelayedRegion(time=671873, expired=True, encoding=auto, regions=[R(12, 8, 400, 387)], options={})
2023-06-09 11:39:31,272 acknowledge_changes() xshm handle=XShmWrapper(0xe000a4 - 424x454), damage handle=4194358
2023-06-09 11:39:31,272 invalidating named pixmap, contents handle=PixmapWrapper(0x40003c, 424, 454)
2023-06-09 11:39:31,272 send_delayed_regions: using full frame for webp encoding of 400x387
2023-06-09 11:39:31,272 refreshing named pixmap
2023-06-09 11:39:31,273 scroll detection took 0ms
2023-06-09 11:39:31,273 make_data_packet: image=XShmImageWrapper(BGRX: 0, 0, 424, 454), damage data: (3, 0, 0, 424, 454, 'webp')
2023-06-09 11:39:31,278 compress:   4.5ms for  424x454  pixels at    0,0    for wid=3     using      webp with ratio   5.1%  (  752KB to    38KB), sequence     5, client_options={'rgb_format': 'BGRX', 'quality': 100, 'encoder': 'webp'}, options={'quality': 100, 'speed': 80, 'rgb_formats': ('RGB', 'RGBX', 'RGBA'), 'lz4': True, 'content-type': 'text', 'cuda-device-context': None}
2023-06-09 11:39:31,278 auto refresh:  webp screen update (actual quality=100, lossy=False), lossless - nothing to do (region=rectangle(0, 0, 424, 454), refresh regions=[], exclude=None)
2023-06-09 11:39:31,280 Warning: client cannot find window 3
```
If I do not kill the IDE, the "client cannot find window 3" repeats in a loop.

Creating a project and changing to the full IDE screen works fine, as does some other applications I've tested (chrome, firefox, konsole).
Could it be to do with a beta version mismatch between the client and server?

**Comment by totaam** [comment]
> Could it be to do with a beta version mismatch between the client and server?

No.
This is a bug, which has already been fixed: a0f26bdb9197e40cb2baf296fdf25566201ae7d3

Unfortunately, I cannot make new beta builds at the moment, so use an older build or 4.4.5.

**Comment by totaam** [comment]
@jzyrobert Please try 5.0-r33655 or later from https://xpra.org/beta/windows/
